docs/api/1.x/authentication.rst

```diff
@@ -356,3 +361,24 @@ The ``parseToken()`` function scans the location hash to read the Identity Provi
     }
 
 Check out the :github:`full demo source code <leplatrem/kinto-oidc-demo>`.
+
+
+Example of 0Auth dance redirections
```

Comment:
OAuth or Auth0?

Reply:
Good catch. It's OAuth, but it's confusing anyway, we should simply put «login dance»